spaceship.zsh

```diff
@@ -111,7 +111,9 @@ source "$SPACESHIP_ROOT/lib/section.zsh"
 # ------------------------------------------------------------------------------
 
 for section in $(_union $SPACESHIP_PROMPT_ORDER $SPACESHIP_RPROMPT_ORDER); do
-  source "$SPACESHIP_ROOT/sections/$section.zsh"
+  if [ -f "$SPACESHIP_ROOT/sections/$section.zsh" ]; then
```

Comment:
This doesn't feel like the right approach. A typo in a section filename will silently fail. I think this can be avoided by declaring plugins in a separate array.

Reply:
Good catch!
I'll be thinking on this. I like the approach to declare somewhere else which external function are to be used.

Reply:
I've been thinking about exposing the option like `SPACESHIP_PROMPT_PLUGINS`. Thoughts?

Reply:
`SPACESHIP_PROMPT_PLUGINS` SGTM